src/workspace/internal/workspaceactioncontroller.cpp

```diff
@@ -29,7 +29,9 @@ using namespace mu::actions;
 void WorkspaceActionController::init()
 {
     dispatcher()->reg(this, "select-workspace", this, &WorkspaceActionController::selectWorkspace);
+    dispatcher()->reg(this, "edit-workspace", this, &WorkspaceActionController::editCurrentWorkspace);
     dispatcher()->reg(this, "configure-workspaces", this, &WorkspaceActionController::openConfigureWorkspacesDialog);
+    dispatcher()->reg(this, "reset-workspace", [this]() { setCurrentWorkspaceName(DEFAULT_WORKSPACE_NAME); });
```

Comment:
As I explained in https://github.com/musescore/MuseScore/issues/12658#issuecomment-1215150531, this will not really reset the workspace... not sure what to do about that.

Reply:
How about basing the default workspace of a "Default workspace template" not exposed to users. But we can call it the default workspace since for the users its the workspace that they use (by default).

So, whenever we want to reset it to default workspace, we just delete the default workspace and create a new one based off of the template.

Reply:
Actually 'reset workspace' doesn't work for me in MU3 at all. 
So I don't know how it is should work. 

In the beginning, I thought that this should reset the current workspace to default state.
But here I see that we are resetting the selection to the default workspace, not the state